docs/_docs/user-guide/eldritch.md

```diff
@@ -312,6 +319,24 @@ crypto.to_json({"foo": "bar"})
 "{\"foo\":\"bar\"}"
 ```
 
+### crypto.md5
```

Comment:
Remove this, this is not a function we have implemented. Please refer to eldritch-libcrypto/lib.rs for the list of functions.